### ISSUE TITLE
Training: learning rate experiments

### ISSUE THREAD

**Issue by inakbk** [mateuszmatu/OceanAI#151]
EDIT:
This is the best docs to read about the scheduler:
https://timm.fast.ai/SGDR#CosineLRScheduler

check out the scheduler

Reading/links:

https://anemoi.readthedocs.io/projects/training/en/latest/modules/train.html#anemoi.training.train.forecaster.forecaster.GraphForecaster.lr_scheduler_step

https://medium.com/@utkrisht14/cosine-learning-rate-schedulers-in-pytorch-486d8717d541

https://github.com/ecmwf/anemoi-core/blob/3a1922b256c1c83af364acf62f37a81d272740f0/training/src/anemoi/training/train/forecaster/forecaster.py#L563

https://github.com/ecmwf/anemoi-core/blob/3a1922b256c1c83af364acf62f37a81d272740f0/training/docs/user-guide/training.rst#L415

https://github.com/huggingface/pytorch-image-models/blob/main/timm/scheduler/cosine_lr.py
